ue5osc/osc_dispatcher.py

```diff
@@ -0,0 +1,63 @@
+from time import sleep
+from typing import Any, List
+
+from pythonosc.dispatcher import Dispatcher
+
+
+class OSCMessageReceiver:
+    def __init__(self):
+        self.values = None
+        self.dispatcher = Dispatcher()
+
+        # Map commands to the set_filter method
+        self.dispatcher.map("/location", self.handle_location)
+        self.dispatcher.map("/rotation", self.handle_rotation)
+        self.dispatcher.map("/project", self.handle_project)
+        self.dispatcher.set_default_handler(self.handle_invalid_command)
+
+    def handle_location(
+        self, address: str, *args: List[Any]
+    ) -> tuple[float, float, float]:
+        # Logic to handle location path
+
+        # This check is necessary
+        if address == "/location":
+            # Split the string argument into three float values
+            values = args[0].split(",")
+            try:
+                x, y, z = map(float, values)
+            except ValueError or len(x, y, z == 3):
+                return
```

Comment:
I wonder if we should let this exception get thrown and make the caller handle it? Otherwise, this will return nothing and we'll get an error anyway since we expected three floats.

Reply:
I'm not quite sure, what letting the caller handle it entails to.